tests/test_security.py

```diff
@@ -90,7 +90,7 @@ def test_ray_cluster_with_security_context(self):
         ray_cluster_add_event = RayClusterAddCREvent(
             custom_resource_object = context['cr'],
             rulesets = [],
-            timeout = 90,
+            timeout = 180,
```

Comment:
I found the RayClusterAddCREvent flaky due to its short timeout. Double it to 180 seconds to reduce flakiness.